src/pages-and-resources/discussions/app-config-form/apps/shared/ExternalDocumentations.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  Hyperlink, MailtoLink,
+} from '@edx/paragon';
+
+import messages from '../lti/messages';
+
+function ExternalDocumentations({
+  documentationUrls,
+  intl,
+  title,
+}) {
+  const appInstructions = Object.keys(documentationUrls);
+  return (
+    <div className="pt-4">
+      <h4>{intl.formatMessage(messages.linkTextHeading)}</h4>
+      <div className="small text-muted">
+        {appInstructions.map((instructionType) => (
+          documentationUrls[instructionType]
+          && instructionType !== 'contactEmail'
+          && (
+            <div key={instructionType}>
+              <FormattedMessage
+                {...messages.linkText}
+                values={{
+                  link: (
+                    <Hyperlink
+                      destination={documentationUrls[instructionType]}
+                    >
+                      <FormattedMessage
+                        {...messages[instructionType]}
```

Comment:
Can you try inline translation here? its easier to read.

Reply:
we have dynamic data which needs to be rendered with a link argument hence I have added this instead of inline messaging.

Reply:
one of the link text contains the title of the app in a sentence, I need to add that as a parameter to the translation that's why I have used this. but I got one more issue in code that upper <FormattedMessage > tag is not needed hence removing that.